### PR TITLE
DRY code for CustomView

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

**Comment by bastianallgeier** [comment]
Ahhh, now I get what the immediate option does. That's pretty fantastic!

**Comment by nilshoerrmann** [comment]
Is it "inmediate" or "immediate“?

File: panel/src/components/Views/CustomView.vue [hunk 1]
```diff
@@ -18,12 +18,12 @@ export default {
     })
   },
   watch: {
-    plugin() {
-      this.$store.dispatch("view", this.plugin);
+    plugin: {
+      handler: () => {
+        this.$store.dispatch("view", this.plugin);
+      },
+      inmediate: true
```
Comment (bvdputte) [inline panel/src/components/Views/CustomView.vue:25]: There's probably a typo? Should be `immediate` (now it's `inmediate`)?

Comment (distantnative) [reply panel/src/components/Views/CustomView.vue:25]: Indeed, thanks for spotting it. Never code after talking to your Spanish-speking SO :D

**Pull request merged**